app/src/main/AndroidManifest.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.androidchallenge">
-
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <application
+        android:usesCleartextTraffic="true"
```

Comment:
That's okay for the challenge (it'd probably not work without it anyway) but, keep in mind that this enables HTTP communication which is not safe comparing to the default HTTPS.